### PR TITLE
resolved issues #45,#52,#53,#54

### LINKED ISSUE #45: (1) Clutter
## Description
`Level 1`
> scrolling is s o o o o much effort. move the zodiac table and skeleton gifs from the home page to the about page so you can click there instead of having scroll. this also means people can focus more on baby yoda.

### LINKED ISSUE #52: (1) Greetings Earthling
**Description**
Level 1
Format the home page such that there is a bold title above the greeting. The title and greeting should be on separate lines and taken from strings.json
(Title: **HackRU Reactment Challenge**
Greeting: Welcome to Reactment, RnD's Interview Sandbox!! Feel Free to Explore! Reactment is not HTML. #Im19andThisIsDeep.)

### LINKED ISSUE #53: (2) The Krabby Patty Formula
**Description**
Level 2
Hide an image of a Krabby Patty somewhere on one of the pages that prints “Are you feeling it now Mr. Krabs?!?” into the console when clicked on.

### LINKED ISSUE #54: (2) Bland Brand Land
**Description**
Level 2
Display the HackRU logo in a header above the Navbar. Add an animation to it when you load the page!

### PR DESCRIPTION
**Pull request by DaiJunyan**
#45, #52, resolves #53, resolves #54

### CODE REVIEW

File: challenge/src/components/Pages/About.jsx [hunk 1]
```diff
@@ -71,11 +74,14 @@ class About extends React.Component {
             <div style={{ backgroundColor: "whitesmoke" }}>
               <br />
             </div>
-            <center>
```
Comment (mjrb) [inline challenge/src/components/Pages/About.jsx:74]: you shouldn't delete this code, it breaks the thanos component.

File: challenge/src/components/Pages/About.jsx [hunk 2]
```diff
@@ -51,7 +54,7 @@ class About extends React.Component {
             </div>
             <div
               style={{
-                backgroundColor: "green",
```
Comment (mjrb) [inline challenge/src/components/Pages/About.jsx:54]: this change isn't necessary `green` will display the same color as `#00ff00` i believe. this was part of issue #66 which some one else also claimed

File: challenge/src/components/Pages/About.jsx [hunk 3]
```diff
@@ -1,5 +1,8 @@
 import React from 'react'
-import JojoStand from '../JojoStand'
+import RandomImage from '../RandomImage'
+import Table from '../Table'
+import GIF from "../Gif"
```
Comment (mjrb) [inline challenge/src/components/Pages/About.jsx:4]: don't need this imports, the table and gif thing will be fixed in another pull request #70

File: challenge/src/components/Pages/About.jsx [hunk 4]
```diff
@@ -71,11 +74,14 @@ class About extends React.Component {
             <div style={{ backgroundColor: "whitesmoke" }}>
               <br />
             </div>
-            <center>
-              <button style={{ margin: "20px", padding:"10px"}} onClick={this.snap}>
-                Thanos Snap
-              </button>
-            </center>
+            <Table/>
```
Comment (mjrb) [inline challenge/src/components/Pages/About.jsx:77]: similarly this is also going to be fixed by a different pull request #70. this change doesn't need to be here

File: challenge/src/components/Pages/Sponsors.jsx [hunk 5]
```diff
@@ -21,7 +21,7 @@ const Sponsors = (props) => {
             </div>
             <div
               style={{
-                backgroundColor: "green",
```
Comment (mjrb) [inline challenge/src/components/Pages/Sponsors.jsx:24]: this change is also not needed

File: challenge/src/components/Pages/Home.jsx [hunk 6]
```diff
@@ -15,6 +14,9 @@ const Home = (props) => {
               style={{ width: "100%", marginTop: "4rem", textAlign: "center" }}
             >
               <Fade left cascade>
+                <b>
+                  {strings.title}
```
Comment (mjrb) [inline challenge/src/components/Pages/Home.jsx:18]: this doesn't need to be added #52 was claimed by someone else

File: challenge/src/components/Pages/Home.jsx [hunk 7]
```diff
@@ -3,9 +3,8 @@ import BabyYoda from "../../mandalorian/BabyYoda";
 import NumberLogger from "../../NumberLogger";
 import strings from "../../strings.json";
 import Fade from "react-reveal/Fade";
-import Table from '../Table'
-import GIF from "../Gif"
```
Comment (mjrb) [inline challenge/src/components/Pages/Home.jsx:7]: these imports should be added back

File: challenge/src/components/Pages/Home.jsx [hunk 8]
```diff
@@ -26,16 +28,9 @@ const Home = (props) => {
                 <NumberLogger />
               </Fade>
             </div>
-              <MonthAndYear />
-              <BabyYoda />
-              <Table/>
-              <div style={{marginTop: "1%", marginBottom: "1%", display: "flex", alignItems: "center", justifyContent: "center"}}>
-                <GIF gifsrc="https://media.giphy.com/media/J0WeVOLjuqW2I/giphy.gif"/>
-                <GIF gifsrc="https://media.giphy.com/media/J0WeVOLjuqW2I/giphy.gif"/>
-                <GIF gifsrc="https://media.giphy.com/media/J0WeVOLjuqW2I/giphy.gif"/>
```
Comment (mjrb) [inline challenge/src/components/Pages/Home.jsx:35]: you can leave this in because its going to be fixed in another PR #70

**Review by mjrb — CHANGES_REQUESTED** [review]
@DaiJunyan there isn't really anything wrong with the code itself, and we always appreciate people having fun and doing multiple issues. However you seem to have missed a very important step in the instructions which was self-assigning/claiming whatever issues you wanted to work on in github. because of this you have changes that resolve #45 and #52 which are currently being resolved by other people. you also seem to have some changes that may be related to #66. this is an issue because when multiple people try to do the same task it makes it hard to give credit for people work, and for the people who did the issue first it kinda feels bad to have someone else redo your code, even though it was likely just a miscommunication.

I appreciate your work but please assign #53 and #54 to yourself under the issues tab in github, and remove any changes related to #45 #53 and #66 from this pull request

**Comment by DaiJunyan** [comment]
I am so sorry, I thought each one just does his/her own version. I will assign issues to myself and remove changes to issues that are being resolved by others.

**Comment by mjrb** [comment]
no problem, don't worry to much. hackru is all about learning and growth. don't be afraid to reach out and ask questions if instructions ever seem unclear
